circle.yml

```diff
@@ -66,9 +66,8 @@ jobs:
       #
       - run: ls -la
       - run: npm run deps
-      - run: npm run lint
+      - run: npm run lint -- --max-warnings=0
```

Comment:
why `--max-warning=0`?

Reply:
This reproduces the current flow when removing `npm run stop-only` - any `.only` causes the lint check to fail.

The question inside is why make `no-only-tests` a warning rather than an error. This is from experience with `npm start` on other projects, where lint errors cause a live reload to abort, but warnings allow it to complete. So, may be less relevant here but have found it helps combining TDD with in-browser development.